packages/eslint-config-xmtp-web/index.js

```diff
@@ -105,6 +105,7 @@ module.exports = {
             peerDependencies: false,
           },
         ],
+        "@typescript-eslint/no-var-requires": "off",
```

Comment:
why do we need to add this rule? just curious

Reply:
some config files may be commonjs and use `require` for imports that are then assigned to a variable using `var` or `const`.